options.go

```diff
@@ -94,7 +94,7 @@ func NewOptions() *Options {
 		PassBasicAuth:       true,
 		PassAccessToken:     false,
 		PassHostHeader:      true,
-		PassRolesHeader:     true,
```

Comment:
Nice catch. I had changed the default flagSet in main.go and neglected to change this as well.